scripts/coverage.sh

```diff
@@ -1,8 +1,15 @@
 #! /bin/bash
 set -e
+
+mkdir -p /var/lib/softhsm/tokens
+softhsm2-util --init-token --slot 0 --label "Virtual token" --pin 1234 --so-pin 1234
+softhsm2-util --import ./src/tests/test_data/implicit/pkey.pem --label "pkey" --id 02 --slot 0 --pin 1234
+openssl x509 -outform der -in ./src/tests/test_data/implicit/client.pem -out ./src/tests/test_data/implicit/client.der
+pkcs11-tool --module=/usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so --id 1 --write-object ./src/tests/test_data/implicit/client.der --type cert --login --pin 1234
+
 mkdir -p build-coverage
 cd build-coverage
-cmake -DBUILD_GENIVI=ON -DBUILD_OSTREE=ON -DBUILD_WITH_CODE_COVERAGE=ON ../src
+cmake -DBUILD_GENIVI=ON -DBUILD_OSTREE=ON -DBUILD_WITH_CODE_COVERAGE=ON -DTEST_PKCS11_ENGINE_PATH="/usr/lib/engines/engine_pkcs11.so" -DTEST_PKCS11_MODULE_PATH="/usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so" ../src
```

Comment:
in newer versions(from yakkety) path "/usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so" has been changed to "/usr/lib/softhsm/libsofthsm2.so"

Reply:
Yes, we should change it when we move to yakkety. Should work with current dockerfile though.